gradle.properties

```diff
@@ -3,7 +3,7 @@ nexusUser=invalid
 nexusPassword=invalid
 
 # IntelliJ Platform Properties -> https://plugins.jetbrains.com/docs/intellij/tools-gradle-intellij-plugin.html#configuration-intellij-extension
-ideaVersion=2024.2
+ideaVersion=2024.3
```

Comment:
please also update the ci to validate against this version ( i know it's this one used in OS-matrix build, but this will provide us verifier reports )